src/util/constants.ts

```diff
@@ -1,26 +1,34 @@
-
 export enum SessionStorageKeys {
-    KeyCloakToken = 'KEYCLOAK_TOKEN',
-    ApiConfigKey = 'AUTH_API_CONFIG',
-    LaunchDarklyFlags = 'LD_FLAGS',
-    ExtraProvincialUser = 'EXTRAPROVINCIAL_USER',
-    SessionSynced = 'SESSION_SYNCED',
-    AuthApiUrl = 'AUTH_API_URL',
-    AuthWebUrl = 'AUTH_WEB_URL',
-    StatusApiUrl = 'STATUS_API_URL'
+  KeyCloakToken = 'KEYCLOAK_TOKEN',
+  ApiConfigKey = 'AUTH_API_CONFIG',
+  LaunchDarklyFlags = 'LD_FLAGS',
+  ExtraProvincialUser = 'EXTRAPROVINCIAL_USER',
+  SessionSynced = 'SESSION_SYNCED',
+  AuthApiUrl = 'AUTH_API_URL',
+  AuthWebUrl = 'AUTH_WEB_URL',
+  StatusApiUrl = 'STATUS_API_URL'
 }
 
 export enum DateFilterCodes {
-    TODAY = 'TODAY',
-    YESTERDAY = 'YESTERDAY',
-    LASTWEEK = 'LASTWEEK',
-    LASTMONTH = 'LASTMONTH',
-    CUSTOMRANGE = 'CUSTOMRANGE'
+  TODAY = 'TODAY',
+  YESTERDAY = 'YESTERDAY',
+  LASTWEEK = 'LASTWEEK',
+  LASTMONTH = 'LASTMONTH',
+  CUSTOMRANGE = 'CUSTOMRANGE'
 }
 
 export enum PaymentMethods {
-    CASH = 'CASH',
-    CHEQUE = 'CHEQUE'
+  CASH = 'CASH',
+  CHEQUE = 'CHEQUE'
+}
+
+export enum SlipStatus {
+  ACTIVE = 'ACTIVE',
+  COMPLETE = 'COMPLETE',
+  BOUNCED='BOUNCED',
+  NSF = 'NSF',
+  REFUND ='REFUND',
+  LAST_SERVICE='LAST SERVICE'
```

Comment:
I think LAST_SERVICE value can be same as the key. Reason being UI will be always displaying the description from the backend ?